pypyr-ops/Dockerfile

```diff
@@ -12,14 +12,24 @@ ARG terraformversion=0.11.8
 
 ARG awsversion=1.16.37
 
-RUN  apt-get update && apt-get install -y apt-transport-https; \
+RUN  apt-get update && apt-get install -y apt-transport-https \
```

Comment:
could you amend this big run to use && in between commmands rather than ;

```
RUN set -ex \
  && dothing \
  && doanotherthing \
  && doyetanotherthing
```
I've got a suspicion there are a couple of errors here that are getting hidden.

Reply:
Done